data/videos.yml

```diff
@@ -3693,3 +3693,30 @@ Seattle Xcoders February 2016:
     title: Continuous Integrations for iOS Apps
     tags: [xcode, build, continuous integration]
     vimeo: 158290425
+NSLondon February 2016:
+  - language: English
+    speakers: [Raimon Lapuente, Gergely Orosz]
+    title: "Sorry Objc: It’s not you, it’s me"
```

Comment:
Sorry for the late reply @douglashill !

I think we should probably strip those characters when we generate the page names...

The code for that is in `source/lib/video_helpers.rb`. I think it's line 24. Do you have time to update the PR for this?